frontend/test/specs/institution/allInstitutionsControllerSpec.js

```diff
@@ -128,4 +138,62 @@
             expect(institutionService.getNextInstitutions).toHaveBeenCalled();
         });
     });
+
+    describe('checkSeenInstitutions()', () => {
+        it('should call getItem and setItem', () => {
+            spyOn(localforage, 'setItem');
+            spyOn(localforage, 'getItem').and.callFake(() => {
```

Comment:
You can create a callFake function in the outer scope, so it is not necessary to define it for each test that is going to use it.